Remotion/ObjectBinding/Web/UI/Controls/BocReferenceValueImplementation/Rendering/BocAutoCompleteReferenceValueRenderer.cs

```diff
@@ -375,7 +364,17 @@ private TextBox GetTextBox (BocRenderingContext<IBocAutoCompleteReferenceValue>
       textBox.Page = renderingContext.Control.Page!.WrappedInstance;
       textBox.ApplyStyle(renderingContext.Control.CommonStyle);
       renderingContext.Control.TextBoxStyle.ApplyStyle(textBox);
+
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.Combobox);
       textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaAutoComplete, HtmlAriaAutoCompleteAttributeValue.Both);
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaExpanded, HtmlAriaExpandedAttributeValue.False);
+      textBox.Attributes.Add(HtmlTextWriterAttribute2.AriaHasPopup, HtmlAriaHasPopupAttributeValue.True);
```

Comment:
why did you change this from a listbox to a boolean value?

Reply:
yeah, no. I believe there's still the four options available and it's possible `true` was also always an option but "listbox" is more descriptive about the intent. 

> Elements with the role combobox have an implicit [aria-haspopup](https://www.w3.org/TR/wai-aria-1.2/#aria-haspopup) value of listbox. If the combobox popup element has a role other than [listbox](https://www.w3.org/TR/wai-aria-1.2/#listbox), authors MUST specify a value for [aria-haspopup](https://www.w3.org/TR/wai-aria-1.2/#aria-haspopup) that corresponds to the role of its popup.

So, it's actually redundant but specifying it makes things explicit. I like explicit. Also, maybe one day we have a grid in there or a tree. then we need the other options, too :)

Reading further:
> To provide backward compatibility with ARIA 1.0 content, user agents MUST treat an `aria-haspopup` value of `true` as equivalent to a value of `menu`.

We do not want `menu`.